lesson17/js/script.js

```diff
@@ -1,117 +1,124 @@
-// // task 1
+// Homework ------------------------------------------------------------
 
-// let a = 1;
-// let b = 1;
-// let c = ++a;
-// let d = b++;
+// Task 1 ------------------------------------------------------------
 
-// console.log(a, b, c, d);
+const strTrue = 'true';
+const isFalse = false;
+const NUMBER_17 = 17;
+const und = undefined;
+const empty = null;
 
-// // task 2
+console.log(typeof(strTrue),
+            typeof(isFalse),
+            typeof(NUMBER_17),
+            typeof(und),
+            typeof(empty));
```

Comment:
Рекомендуют это записать так для читаемости кода
```suggestion
console.log(
   typeof(strTrue),
   typeof(isFalse),
   typeof(NUMBER_17),
   typeof(und),
   typeof(empty),
);
```